lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -286,6 +286,7 @@ class CacheHeaders extends Audit {
       return {
         score,
         numericValue: totalWastedBytes,
+        numericUnit: 'bytes',
         displayValue: str_(UIStrings.displayValue, {itemCount: results.length}),
```

Comment:
we have a bunch of places where the `displayValue` is *not* `formatter.format(numericValue, {unit: numericUnit})`, so it may be time to finally add a comment to the `displayValue` type declaration that it's whatever the audit wants to display for itself (leaving aside that the perf-category-renderer ignores that and only uses the displayValue for the hover text :)